step-definitions/standard-ui/button.js

```diff
@@ -25,6 +25,21 @@ When('I click on button {string}',
   }
 );
 
+/**
```

Comment:
Avoid selecting by ID in cucumber. Its purpose are easy to understand tests even for DAUs. IDs are invisible to them and hence hard to use.

Reply:
but if buttons exist with the same name, this can't be avoided

Reply:
I see :)